src/htm/types/Serializable.hpp

```diff
@@ -95,7 +97,7 @@ typedef enum {BINARY, PORTABLE, JSON, XML} SerializableFormat;
 // Archive?
 // see: http://uscilab.github.io/cereal/serialization_archives.html
 // The Cereal package passes the varients of the Archive package around to act sort of
-// like ostream and such.  Archive acts like a base class but in fact it is nothing 
+// like ostream and such.  Archive acts like a base class but in fact it is nothing
```

Comment:
nit, please try to avoid the editor whitespace changes, unless necessary. Makes reviewing the PR (or any further diffs) more difficult.

Reply:
> avoid the editor whitespace changes

This was just trying to get it to build again.
Perhaps there is a better way....

Reply:
>  to build again.
Perhaps there is a better way....

you can Close & Reopen the PR, it'll retrigger the CI builds.